src/blog/2022/09/flowforge-09-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: FlowForge 0.9 released
+subtitle: Suspended projects, login with email and Team Types
+description: Suspended projects, login with email and Team Types
+date: 2022-09-01 12:00:00.0
+authors: ["sam-machin"]
+video: BBNzFInc_2E
```

Comment:
```suggestion
video: d23Pmyc0k7I
```

Reply:
The video is currently unlisted - will need to be made public when we publish this post.